### ISSUE TITLE
Decide on a continuous integration (CI) system.

### ISSUE THREAD

**Issue by deanberris** [cpp-netlib/netlib#3]
Pick one of:

- Travis
- CircleCI
- Google Cloud Build
- Something Else

for the CI system.

**Comment by glynos** [comment]
Another candidate might be AppVeyor:

https://www.appveyor.com/

**Comment by deanberris** [comment]
Thanks -- though I don't see macOS support in the AppVeyor docs, which Travis and CircleCI have, but I don't see Windows in either Travis nor CircleCI. 😕 

The worst case scenario here might be that we use one for Linux/macOS build/testing/deployment and one for Windows. Looks like AppVeyor is the only one that supports Windows from this list.

How hard would that be to set-up?

**Comment by glynos** [comment]
I used AppVeyor for CI builds with the URL library for Windows, and it was as easy as Travis to set up.

I felt that using a combination of Travis and AppVeyor gave me important feedback and helped me catch cross-platform bugs as soon as I had committed, and I felt that the overhead (admittedly on a small library) was not a lot given the benefit.

**Comment by brown2rl** [comment]
Where do you all want to set up the CI server?  Looks like between USA, Australia, and Belgium might be somewhere in east Asia like Russia, Philippines, or Japan.

**Comment by deanberris** [comment]
We use the free CI automation runners that the CI providers allow open source projects to use, if we can help it. These are run on public clouds and don't need to be hosted in a specific region. Was there a specific concern ypu had, @brown2rl on the location of the CI server(s)?

**Comment by brown2rl** [comment]
No, thanks. Looks like it's Travis + AppVeyor, if you've had success with that.

**Comment by brown2rl** [comment]
I attempted to set up Travis + AppVeyor, however it's a job for the repo owner or organization member. Would I be able to to be added to the organization? Or can we make a new one! Thanks!

**Comment by glynos** [comment]
@brown2rl I have configured [Travis](https://travis-ci.org/cpp-netlib/netlib), [AppVeyor](https://ci.appveyor.com/project/glynos/netlib) and [CircleCI](https://circleci.com/gh/cpp-netlib/netlib) for the cpp-netlib/netlib repo.

@deanberris Let me know if I still need to provide access for any of these, if you can't access them.

**Comment by brown2rl** [comment]
@glynos great job!

**Comment by AngleNet** [comment]
Maybe we should integrate tester, linter to CI? If it is not done yet, I can help.
